docs/querying/multi-value-dimensions.md

```diff
@@ -23,27 +23,52 @@ title: "Multi-value dimensions"
   -->
 
 
-Apache Druid supports "multi-value" string dimensions. These are generated when an input field contains an
-array of values instead of a single value (e.g. JSON arrays, or a TSV field containing one or more `listDelimiter`
-characters). By default Druid ingests the values in alphabetical order, see [Dimension Objects](../ingestion/index.md#dimension-objects) for configuration.
+Apache Druid supports "multi-value" string dimensions, which result from input fields that contain an
+array of values instead of a single value. 
```

Comment:
Might be good to have an example input -> Druid here (setup for tags example later?)